source/sphinx_extensions/plugin_directory/templates/available.rst

```diff
@@ -7,5 +7,10 @@ QIIME 2 microbiome analysis functionality is made available to users via plugins
    :maxdepth: 3
 
    {% for name, _ in plugins|dictsort %}
+   {% if _.short_description %}
```

Comment:
This is minor: can you change the variable name `_` to something more descriptive such as `plugin`? The original code didn't use the values of the dictionary while iterating, so we assigned to a throwaway variable named `_` (this is just convention in Python and doesn't have any special meaning). Now that the values of the dict are being used it's a good idea to give the variable a name.